src/legacy/core_plugins/vis_type_metric/public/components/metric_vis_params/metric_vis_params.js

```diff
@@ -1,86 +0,0 @@
-/*
- * Licensed to Elasticsearch B.V. under one or more contributor
- * license agreements. See the NOTICE file distributed with
- * this work for additional information regarding copyright
- * ownership. Elasticsearch B.V. licenses this file to you under
- * the Apache License, Version 2.0 (the "License"); you may
- * not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-
-import _ from 'lodash';
-import { i18n } from '@kbn/i18n';
-import metricVisParamsTemplate from './metric_vis_params.html';
-
-export function MetricVisParams() {
-  return {
-    restrict: 'E',
-    template: metricVisParamsTemplate,
-    replace: true,
-    link: function ($scope) {
-      $scope.collections = $scope.vis.type.editorConfig.collections;
-      $scope.showColorRange = true;
-
-      $scope.$watch('editorState.params.metric.metricColorMode', newValue => {
```

Comment:
Where was this logic moved to?

Reply:
The logic is duplicated in `metric_vis_fn.ts` and passes into the config after applying changes: 
![image](https://user-images.githubusercontent.com/31325372/65975211-f4739480-e476-11e9-9a85-4e3072a7fefe.png)

So I think it is redundant to watch these params in the editor. 
This also calculates in visualization controller `metric_vis_controller.js` to reflect on UI:
![image](https://user-images.githubusercontent.com/31325372/65975162-e160c480-e476-11e9-86bf-4bb2ad94f444.png)